src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -2279,8 +2248,8 @@ public void RemoveAllTransactionsWithSyncEnabledSyncsAfterRemoval()
             var walletManager = new Mock<IWalletManager>();
             var walletSyncManager = new Mock<IWalletSyncManager>();
             walletManager.Setup(manager => manager.RemoveAllTransactions(walletName)).Returns(resultModel);
-            walletManager.Setup(manager => manager.GetWallet(walletName)).Returns(wallet);
-            walletSyncManager.Setup(manager => manager.SyncFromHeight(It.IsAny<int>()));
+            //walletManager.Setup(manager => manager.GetWalletByName(walletName)).Returns(wallet);
```

Comment:
Please remove or add comment on why this commented out.

Reply:
Removed. Not required.